lib/Option/SanitizerOptions.cpp

```diff
@@ -185,6 +185,46 @@ OptionSet<SanitizerKind> swift::parseSanitizerArgValues(
   return sanitizerSet;
 }
 
+OptionSet<SanitizerKind> swift::parseSanitizerRecoverArgValues(
```

Comment:
We have `parseSanitizerRecoverArgValues` and `parseSanitizerArgValues`, both needing to parse a list of sanitizers and returning an `OptionSet<SanitizerKind>`.  Do you think there is an opportunity to share some code here?

Reply:
While there is some similarity I'm not convinced there's enough code there to justify refactoring. In the future we might also replace `SanitizerKind` with something else (e.g. `SanitizerRecoverKind`) because we might might want to allow fine grained control over the kinds of behaviour that allows execution to continue rather than trapping.

I didn't make a `SanitizerRecoverKind` because I only really need ASan right now and re-using the `SanitizerKind` enum was extremely convenient.